FiguresInheritance/Models/Square.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace FiguresInheritance.Models
+{
+    internal class Square : LineSegment // Квадрат
+    {
+        protected float Width { get; set; }
+        protected float Perimeter { get; set; }
+        protected float Area { get; set; }  // Площадь
```

Comment:
I hope you understand
![image](https://user-images.githubusercontent.com/22522346/158749609-310bf43a-c214-4c6e-85a4-2d7cb76acb7c.png)